python/py-pick/Portfile

```diff
@@ -31,7 +31,5 @@ if {${name} ne ${subport}} {
     depends_test-append \
                     port:py${python.version}-pytest
 
-    test.run        yes
-
```

Comment:
Kirill, just to confirm, tests will now be disabled for this port?

Reply:
@mascguy unfortnently the new release hasn't got tests as part of pip distributive anymore :(